api-docs/v2/getting-started/install-CLI-client.rst

```diff
@@ -23,7 +23,7 @@ underneath your current directory.
       prod:
         auth:
           auth_url: https://identity.api.rackspacecloud.com/v2.0/
-          project_id: 123456
+          project_id: <RACKSPACE_CLOUD_TENANT_ID>
```

Comment:
Abort!!  I just merged as it…  I was remembering the OLD policy…Tenant ID is right!

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting – the #1 Managed Cloud Company
DevDocs on Developer Experience Team
Office: (210) 312-8193   Cell: (210) 364-1060
cat.lookabaugh@rackspace.com

We should not write so that it is possible for the reader to understand us, but so that it is impossible for him to misunderstand us. - Quintilian, rhetorician (c. 35-100)

Harmony + Learner + Responsibility + Context + Intellection + Arranger + Consistency + Relator + Input + Analytical


From: Sabeen Syed <notifications@github.com<mailto:notifications@github.com>>
Reply-To: rackerlabs/docs-cloud-dns <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, March 1, 2016 at 11:58 AM
To: rackerlabs/docs-cloud-dns <docs-cloud-dns@noreply.github.com<mailto:docs-cloud-dns@noreply.github.com>>
Cc: Cat Lookabaugh <cat.lookabaugh@RACKSPACE.COM<mailto:cat.lookabaugh@RACKSPACE.COM>>
Subject: Re: [docs-cloud-dns] Changed clouds.yaml to include tenant id (#50)


In api-docs/v2/getting-started/install-CLI-client.rst<https://github.com/rackerlabs/docs-cloud-dns/pull/50#discussion_r54606758>:

> @@ -23,7 +23,7 @@ underneath your current directory.
>        prod:
>          auth:
>            auth_url: https://identity.api.rackspacecloud.com/v2.0/
> -          project_id: 123456
> +          project_id: <RACKSPACE_CLOUD_TENANT_ID>


Sounds good - I'll go ahead and make that change.

—
Reply to this email directly or view it on GitHub<https://github.com/rackerlabs/docs-cloud-dns/pull/50/files#r54606758>.